src/shogun/regression/KernelRidgeRegression.h

```diff
@@ -135,17 +122,11 @@ class CKernelRidgeRegression : public CKernelMachine
 	private:
 		void init();
 
-		/** train regression using Gauss-Seidel iterative method
-		 *
-		 * @return whether training was successful
-		 */
-		bool train_machine_gs();
-
-		/** train regression using pinv
+		/** train regression using Eigen-3's LLT(Cholesky decomposition)
```

Comment:
btw can you change the interface a bit.
This method should be protected and just return the alphas.
Then the train method itself should prepare everything, call the method, and then set the returned alphas.
This way, we can change the solver *only* without having to re-write all the checks etc